### ISSUE TITLE
Uncaught ReferenceError: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by tonghoangvu** [tonghoangvu/better-code-block#2]
Got this error after building `dist`, reloading the extension, and opening the popup.

![image](https://user-images.githubusercontent.com/64119341/151756808-777022fd-1411-473d-806d-4eea8199979b.png)

Looks like the problem is related to Babel when transpile async. The error does not appear when loading the extension directly from the source.

**Comment by tonghoangvu** [comment]
This helped me fix the problem, use Solution 3 (for Babel 7).

https://exerror.com/babel-referenceerror-regeneratorruntime-is-not-defined/

But I got another error "Uncaught ReferenceError: require is not defined".

![image](https://user-images.githubusercontent.com/64119341/151757900-a29a729d-b9d8-4cf6-9d7b-044f1d65751b.png)

I tried to google many times, this error can only be fixed by using Webpack or Browserify.
